Code/Scripts/wwmath.h

```diff
@@ -64,9 +64,9 @@
 // (DRM 05/07/01) Temporarily eliminated _fastcall
 // on non-Microsoft compatible compilers. Jani
 // should be replacing this soon.
-#ifndef _MSC_VER
+#ifndef _WIN32
```

Comment:
Maybe just remove the use of fastcall entirely from the code base, it only appears to be used on the ASM version of Inv_Sqrt in wwmath.h anyhow (both locations the file exists in).

Reply:
No more fastcall.